liquibase-core/src/test/groovy/liquibase/database/core/CockroachDatabaseTest.groovy

```diff
@@ -0,0 +1,27 @@
+package liquibase.database.core
```

Comment:
@nvoxland, thank you for adding this excellent set of tests. Do these tests run during the build? I don't see CRDB in the list of checks, but perhaps CRDB tests run in the Postgres bucket?